molo/yourwords/templates/yourwords/thank_you.html

```diff
@@ -13,13 +13,16 @@
   </div>
 
 <div class="block {{competition.thank_you_page.get_parent_page.get_effective_extra_style_hints}}">
-    {% if competition.thank_you_page %}
-      <h3>{{competition.thank_you_page.title}}</h3>
-      {% if competition.thank_you_page.subtitle %}
-        <p>{{competition.thank_you_page.subtitle}}</p>
+  {% load_thank_you_page_for_competition competition as thank_you_pages %}
+    {% if thank_you_pages %}
+    {% for page in thank_you_pages %}
+      <h3>{{page.title}}</h3>
+      {% if page.subtitle %}
+        <p>{{page.subtitle}}</p>
       {% else %}
```

Comment:
We don't need this else block here. If there's not subtitle, we should just not show it.
Also, I think we should be rendering the `body` as well. I expect something like
```
{% if page.subtitle %}<p>{{page.subtitle}}</p>{% endif %}
```
and you're going to need this for the body: [article_page](https://github.com/praekelt/molo/blob/develop/molo/core/cookiecutter/scaffold/%7B%7Bcookiecutter.directory%7D%7D/%7B%7Bcookiecutter.app_name%7D%7D/templates/core/article_page.html#L17-L37)